app/views/fields/edit/_primary_publishing_organisation.html.erb

```diff
@@ -1,3 +1,9 @@
+<% if edition.access_limit.present? %>
+  <%= render "govuk_publishing_components/components/warning_text", {
+    text: t("tags.edit.organisation_warning"),
+  } %>
+<% end %>
+
 <%= render "components/autocomplete", {
   id: "#{tag_field.id}-field",
   name: "tags[#{tag_field.id}][]",
```

Comment:
Can we also look at making the naming consistent with [content fields](https://github.com/alphagov/content-publisher/blob/master/app/views/content/edit/_summary_field.html.erb#L22)? These don't have a `content[]` wrapper, which helps make them 'screen-agnostic'.